src/api.ts

```diff
@@ -15,6 +17,7 @@
   // eslint-disable-next-line prefer-template
   const fullURL = BASE_URL + url + '.json';
 
+  // we add some delay to see how the loader works
   return wait(300)
```

Comment:
The delay in the `wait` function is hardcoded to 300ms. Consider making this a parameter of the `get` function to allow for more flexible usage.